build/containers/Dockerfile.agent

```diff
@@ -9,7 +9,16 @@ RUN echo "Using Rust binaries from ${CROSS_BUILD_TARGET}/${BUILD_TYPE}"
 # Link the container to the Akri repository
 LABEL org.opencontainers.image.source https://github.com/project-akri/akri
 
-RUN apt-get update && apt-get install -y --no-install-recommends libssl-dev openssl && apt-get clean
+RUN apt-get update && apt-get install -y --no-install-recommends libssl-dev openssl curl ca-certificates && apt-get clean
+
 COPY ./target/${CROSS_BUILD_TARGET}/${BUILD_TYPE}/agent /agent
+
+# Install crictl
+RUN VERSION="v1.17.0" && \
```

Comment:
`VERSION` is set, can it be used in the subsequent commands (curl, tar, rm)?

Reply:
Sorry that was a remnant of me trying to do just that. I couldnt get the variable substitution to work like it would in normal bash. I kept getting:
```
/bin/sh: 1: Bad substitution
The command '/bin/sh -c curl -L https://github.com/kubernetes-sigs/cri-tools/releases/download/$VERSION/crictl-${$VERSION}-linux-amd64.tar.gz --output crictl-${$VERSION}-linux-amd64.tar.gz &&    tar zxvf crictl-$VERSION-linux-amd64.tar.gz -C /usr/local/bin &&     rm -f crictl-$VERSION-linux-amd64.tar.gz &&     apt-get remove -y curl ca-certificates && apt-get clean' returned a non-zero code: 2
```
Any idea how to make it work? I also tried with using an `ENV VERSION="v1.17.0"` on the previous line

Reply:
fixed in 879ef84ae58f182de0b3a07c635cf1fee2843e6d key was no brackets or parentheses 🤷